src/app/shared/error404/error404.component.scss

```diff
@@ -0,0 +1,18 @@
+.error {
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    .error__text{
+        text-align: center;
+    }
+    .error__logo {
+        display: flex;
+        justify-content: center;
+        padding: 32px;
+        .error__logo__image{
+            max-width: 274px;
+        }
+    }
+}
```

Comment:
Tutaj można child elementy w BEM zagnieżdżać za pomocą symbolu `&`,  nie trzeba wtedy pisać całej klasy od nowa i wygląda czytelniej.
Tutaj link jak to moża robić: https://medium.com/swlh/sass-bem-nesting-variables-970403f42dd6

Reply:
Poprawione, teraz wszystko jest zagnieżdżone `&`